### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by memani1**
Modified the text for compliance in the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,12 +13,35 @@ The MLPerf Training rules are available at [training\_rules](https://github.com/
 
 The MLPerf HPC specific rules are at [hpc\_training\_rules](https://github.com/mlcommons/training_policies/blob/master/hpc_training_rules.adoc).
 
+## Compliance
+The MLPerf logging package implements logging and compliance-checking utilities
+for MLPerf benchmarks in hpc-1.1.0 branch of the MLPerf logging repository (https://github.com/mlcommons/logging/tree/hpc-1.0-branch).
+These work for the HPC v2.0 submissions as well.
```
Comment (sparticlesteve) [inline README.md:19]: Actually we may have a newer branch later. This should be tentatively fine for folks who want to test things now, though

File: README.md [hunk 2]
```diff
@@ -13,12 +13,35 @@ The MLPerf Training rules are available at [training\_rules](https://github.com/
...
+
+To install and test compliance of your runs/submissions:
+
+```
+# Install the package into your python environment.
+# A development install (-e) is recommended for now so you can pull new updates.
+git clone -b hpc-1.0-branch https://github.com/mlperf-hpc/logging.git mlperf-logging
```
Comment (sparticlesteve) [inline README.md:26]: that's the old v0.7 URL. Should be updated to use https://github.com/mlcommons/logging

File: README.md [hunk 3]
```diff
@@ -13,12 +13,35 @@ The MLPerf Training rules are available at [training\_rules](https://github.com/
...
+# Test a full submission folder
+python3 -m mlperf_logging.package_checker <YOUR SUBMISSION_FOLDER> hpc 1.0.0
+```
+
+There is also a script that tests both compliance and summarizes the results. From the mlperf-logging directory (https://github.com/mlcommons/logging), use
+```
+./scripts/verify_for_v1.0_hpc.sh
+```
+
+ <!--
```
Comment (sparticlesteve) [inline README.md:38]: maybe just remove all the obsolete details.

**Review by sparticlesteve — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (memani1) [reply README.md]: Fixed in 7cf95be6253ad51b5d9b96b78f8757067e50331b

File: README.md (hunk 3 above)
Comment (memani1) [reply README.md]: Fixed in 7cf95be6253ad51b5d9b96b78f8757067e50331b

**Comment by memani1** [comment]
@sparticlesteve The URL is fixed in 7cf95be6253ad51b5d9b96b78f8757067e50331b

**Review by sparticlesteve — APPROVED** [review]
LGTM

**Pull request merged**